Error while checking build/O2/fullCI for 674f18754e3d1d877f12ef4b64ec45c0410eb3b8 at 2024-05-30 02:46:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
/sw/SOURCES/O2/13181-slc8_x86-64/0/Detectors/FOCAL/base/include/FOCALBase/Hit.h:131:21: error: 'emcal' is not a member of 'o2'
/sw/SOURCES/O2/13181-slc8_x86-64/0/Detectors/FOCAL/base/include/FOCALBase/Hit.h:131:31: error: template argument 1 is invalid
/sw/SOURCES/O2/13181-slc8_x86-64/0/Detectors/FOCAL/base/include/FOCALBase/Hit.h:131:70: error: 'emcal' is not a member of 'o2'
/sw/SOURCES/O2/13181-slc8_x86-64/0/Detectors/FOCAL/base/include/FOCALBase/Hit.h:131:80: error: template argument 1 is invalid
/sw/SOURCES/O2/13181-slc8_x86-64/0/Detectors/FOCAL/base/include/FOCALBase/Hit.h:131:21: error: 'emcal' is not a member of 'o2'
/sw/SOURCES/O2/13181-slc8_x86-64/0/Detectors/FOCAL/base/include/FOCALBase/Hit.h:131:31: error: template argument 1 is invalid
/sw/SOURCES/O2/13181-slc8_x86-64/0/Detectors/FOCAL/base/include/FOCALBase/Hit.h:131:70: error: 'emcal' is not a member of 'o2'
/sw/SOURCES/O2/13181-slc8_x86-64/0/Detectors/FOCAL/base/include/FOCALBase/Hit.h:131:80: error: template argument 1 is invalid
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/13181/674f18754e3d1d877f12ef4b64ec45c0410eb3b8/build_O2_fullCI/pretty.html).

--- a/Detectors/FOCAL/base/include/FOCALBase/Hit.h
+++ b/Detectors/FOCAL/base/include/FOCALBase/Hit.h
@@ -128,7 +128,7 @@ std::ostream& operator<<(std::ostream& stream, const Hit& point);
 namespace std
 {
 template <>
-class allocator<o2::emcal::Hit> : public o2::utils::ShmAllocator<o2::focal::Hit>
+class allocator<o2::focal::Hit> : public o2::utils::ShmAllocator<o2::focal::Hit>
 {
 };
 } // namespace std

--- a/Detectors/FOCAL/base/include/FOCALBase/Hit.h
+++ b/Detectors/FOCAL/base/include/FOCALBase/Hit.h
@@ -128,7 +128,7 @@ std::ostream& operator<<(std::ostream& stream, const Hit& point);
 namespace std
 {
 template <>
-class allocator<o2::emcal::Hit> : public o2::utils::ShmAllocator<o2::emcal::Hit>
+class allocator<o2::emcal::Hit> : public o2::utils::ShmAllocator<o2::focal::Hit>
 {
 };
 } // namespace std

--- a/Detectors/FOCAL/base/src/Hit.cxx
+++ b/Detectors/FOCAL/base/src/Hit.cxx
@@ -40,6 +40,7 @@ bool Hit::operator<(const Hit& other) const
 Hit& Hit::operator+=(const Hit& other)
 {
   SetEnergyLoss(GetEnergyLoss() + other.GetEnergyLoss());
+  return *this;
 }
 
 void Hit::printStream(std::ostream& stream) const

